internal/service/environment.go

```diff
@@ -108,6 +108,7 @@ func (e *Environment) DeleteEnvironment(ctx *context.Context, request *environme
 	return err
 }
 
+// UpdateEnvironment updates environment
```

Comment:
do we need this comment. Seems obvious :D

Reply:
github action complains otherwise 🗡️